tests/test_exceptions.py

```diff
@@ -70,13 +70,20 @@ def test_aborter_custom():
 
 def test_exception_repr():
     exc = exceptions.NotFound()
-    assert text_type(exc) == '404: Not Found'
+    assert text_type(exc) == (
+        '404 Not Found: The requested URL was not found '
+        'on the server.  If you entered the URL manually please check your '
+        'spelling and try again.')
     assert repr(exc) == "<NotFound '404: Not Found'>"
 
     exc = exceptions.NotFound('Not There')
-    assert text_type(exc) == '404: Not Found'
+    assert text_type(exc) == '404 Not Found: Not There'
     assert repr(exc) == "<NotFound '404: Not Found'>"
 
+    exc = exceptions.HTTPException('An error message')
+    assert text_type(exc) == 'None Unknown Error: An error message'
```

Comment:
This isn't particularly pretty, could we replace `None` with something like `???` (or omit it entirely)?